libcontainer/configs/config.go

```diff
@@ -320,6 +320,9 @@ func (c Command) Run(s HookState) error {
 		Stdout: &stdout,
 		Stderr: &stderr,
 	}
+	if cmd.Env == nil {
+		cmd.Env = []string{"_GO_DOES_NOT_PROVIDE_A_WAY_TO_CLEAR_ENV="}
```

Comment:
Does `cmd.Env = []string{}` also not work?

Reply:
> Does `cmd.Env = []string{}` also not work?

Looks like [it does][1] :).  Fixed with 5f6ed84c → 96e87700.

[1]: https://play.golang.org/p/UZDaL_Ukiht